apps/subs_web/frontend/webpack.config.js

```diff
@@ -74,7 +89,6 @@ module.exports = {
   },
   plugins: [
     new ExtractTextPlugin({ filename: 'app.bundle.css', allChunks: true }),
-    new webpack.optimize.CommonsChunkPlugin({ name: 'vendor', filename: 'vendor.bundle.js' }),
```

Comment:
What's the reasoning behind removing the vendor bundle?

Reply:
I removed the changes for Webpack 4.

https://webpack.js.org/plugins/commons-chunk-plugin/

"The CommonsChunkPlugin has been removed in webpack v4 legato. To learn how chunks are treated in the latest version, check out the SplitChunksPlugin."